### PR TITLE
[#9470] ignore dark mode on iOS

### PR DESCRIPTION
**Pull request by rasom**
fixes (partially)  #9470

### CODE REVIEW

File: ios/StatusIm/AppDelegate.m [hunk 1]
```diff
@@ -50,6 +50,9 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
   RCTRootView *rootView = [[RCTRootView alloc] initWithBridge:bridge
                                                       moduleName:@"StatusIm"
                                                initialProperties:nil];
+  if (@available(iOS 13.0, *)) {
+      rootView.overrideUserInterfaceStyle = UIUserInterfaceStyleLight;
+  }
```
Comment (jakubgs) [inline ios/StatusIm/AppDelegate.m]: Maybe a comment explaining this hack would be a good idea?

Comment (rasom) [reply ios/StatusIm/AppDelegate.m]: It looks kind of self explanatory to me, what exactly would you like to see there? I will add it.

**Review by flexsurfer — APPROVED** [review]

**Comment by errorists** [comment]
I don't see a difference, Status Bar is still using the white appearance, making it invisible. Keyboards are using the dark appearance, image picker remains all bugged up 😿

File: ios/StatusIm/AppDelegate.m (hunk 1 above)
Comment (jakubgs) [reply ios/StatusIm/AppDelegate.m]: Comments aren't supposed to explain what code does, they are supposed to explain why we want what it does. Why do we want this hack? It's not clear from the code.

**Comment by rasom** [comment]
@errorists the issue with amount on transaction sending is fixed, but it seems that the stuff which is outside application's view is not affected by this change

**Comment by errorists** [comment]
@rasom can you go a layer upward, more global, more root? ;)

File: ios/StatusIm/AppDelegate.m (hunk 1 above)
Comment (rasom) [reply ios/StatusIm/AppDelegate.m]: We will have a link to the original GH issue in the commit, should be sufficient enough imo, but well I can put that link to the code as well if necessary.

Comment (jakubgs) [reply ios/StatusIm/AppDelegate.m]: Yes, usually `git blame` is all you need, but if something is a "temporary solution" as you state in the PR description than I think it's only fair to indicate that with a comment. It's too easy to forget about stuff like that and leave it there without a clear indication that it's a temporary hack. But that's just my preference.

**Comment by rasom** [comment]
it seems that we need to upgrade RN in order to fix status bar https://github.com/facebook/react-native/issues/26619#issuecomment-541143915

**Comment by rasom** [comment]
keyboard should be fixed now, not sure if we can fix image picker

**Comment by rasom** [comment]
I read a bit more about `UIUserInterfaceStyle` and seems that it is a problem only in case if app is built using Xcode 10.*, it should work in our case. 

@errorists please take a look when build will be ready. Keyboard, image picker, transaction amount are fixed, status bar is not. We still need to upgrade RN for status bar, I will try to do this tomorrow.

**Comment by rasom** [comment]
I will merge this one and upgrade RN in a separate PR, it might take some time.

**Pull request merged**